src/Interpreters/Slack/SlackEventInterpreter.php

```diff
@@ -1,12 +1,14 @@
 <?php
 namespace actsmart\actsmart\Interpreters\Slack;
 
-use actsmart\actsmart\Interpreters\BaseInterpreter;
```

Comment:
This is not really document so I understand the confusion but the SlackEventInterpreter is not an Intent interpreter. You can revert all of these changes. It's also not really used anywhere right now. The idea behind the SlackEventInterpreter is that it was going to provide additional context based on an event coming through that other components might use.